src/systemctl/systemctl-show.c

```diff
@@ -1858,6 +1859,41 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 return bus_log_parse_error(r);
 
                         return 1;
+                } else if (contents[0] == SD_BUS_TYPE_STRUCT_BEGIN && streq(name, "OpenFile")) {
+                        char *path, *fdname;
```

Comment:
Please use `const char *`

Reply:
When I do this, I have a compilation warning
```
../src/systemctl/systemctl-show.c:1875:57: warning: initialization discards ‘const’ qualifier from pointer target type [-Wdiscarded-qua]
 1875 |                                                 .path = path,
      |                                                         ^~~~
```